examples/chip-tool/templates/logging/EntryToText.zapt

```diff
@@ -0,0 +1,11 @@
+{{> header}}
+
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app-common/zap-generated/ids/Commands.h>
+
+char const * ClusterIdToText(chip::ClusterId id);
+
+char const * AttributeIdToText(chip::ClusterId cluster, chip::AttributeId id);
+
+char const * CommandIdToText(chip::ClusterId cluster, chip::CommandId id);
```

Comment:
I think we need separate AcceptedCommandIdToText and GeneratedCommandIdToText functions, probably.

Reply:
Done.